benchmark/cublas/main.cpp

```diff
@@ -40,7 +40,7 @@ int main(int argc, char** argv) {
   // Initialize googlebench
   benchmark::Initialize(&argc, argv);
 
-  // Create a sycl blas sb_handle from the queue
+  // Create a portBLAS sb_handle from the queue
```

Comment:
I think this comment is misplaced and can be removed. It was probably introduced by a wrong copy&paste. Can you please remove it?